content/articles/2018-02-forcats-0-3-0.Rmd

```diff
@@ -0,0 +1,48 @@
+---
+title: "forcats 0.3.0"
+slug: forcats-0-3-0
+description: >
+    A new version of forcats is now on CRAN!
+date: 2018-02-20
+photo:
+  url: https://www.pexels.com/photo/photo-of-a-cat-sleeping-on-gray-concrete-bench-757457/
+  author: Nicole Law
+categories: [package]
+---
+
+```{r setup, include = FALSE}
+library(forcats)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are pleased to announce that [forcats 0.3.0](http://forcats.tidyverse.org/) is now on CRAN. 
+
+The forcats package provides a suite of useful tools that solve common problems with factors in R. This version includes updates that resolve several of the "most popular" GitHub issues (as measured by reactions), some of which are highlighted below. For a complete set of changes, please see the [release notes](https://github.com/tidyverse/forcats/releases/tag/v0.3.0).
+
+You can install the latest version with:
+
+```{r, eval = FALSE}
+install.packages("forcats")
+```
+
+### Improvements to NA handling
+
+Closing out one of the most up-voted [issues](https://github.com/tidyverse/forcats/issues/89) in the tidyverse, `as_factor()` and `fct_inorder()` can accept NA levels. 
+
+`fct_lump()` now correctly accounts for NA in input. `lvls_revalue()` preserves NA levels. Missing values, including NAs, can be given an explicit factor level using `fct_explicit_na()`, which ensures that missing values will appear in summaries and plots.
+
+### New features
+
+Consistent with other tidyverse packages, all functions that take `...` now use tidy dots. This means that you can use `!!!` to splice in a list of values, and trailing empty arguments are automatically removed. Additionally, all other arguments gain a `.` prefix in order to avoid unhelpful matching of named arguments.
+
+`fct_lump()` has a new argument, `w`, for weighing value frequencies before lumping them together.
+
+### API changes
+
```

Comment:
We needed to make two backward incompatible changes in order to increase consistency across the tidyverse: